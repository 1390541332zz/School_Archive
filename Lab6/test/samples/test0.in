encrypt i like bacon
decrypt r Orpv yzxlm