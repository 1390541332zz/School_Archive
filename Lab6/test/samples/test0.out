encrypted: r orpv yzxlm
decrypted: i Like bacon
